content/docs/rfcs/14/README.md

```diff
@@ -34,183 +38,227 @@ When sending messages over Waku, there are multiple requirements:
+* The `contentTopic` attribute MUST specify a string identifier that can be used for content-based filtering,
+as desribed in [23/WAKU2-TOPICS](/spec/23/).
 
-* The `meta` attribute, if present, contains an arbitrary application-specific variable-length byte array with a maximum length limit of 64 bytes.
-This attribute can be utilized to convey supplementary details to various Waku protocols, thereby enabling customized processing based on its contents.
+* The `meta` attribute, if present,
+SHOULD contain an arbitrary application-specific variable-length byte array with a maximum length limit of 64 bytes.
+This attribute can be utilized to convey supplementary details to various Waku protocols, thereby
+enabling customized processing based on its contents.
 
-* The `version` attribute, if present, contains a version number to discriminate different types of payload encryption.
+* The `version` attribute, if present,
+SHOULD contain a version number to discriminate different types of `payload` encryption.
 If omitted, the value SHOULD be interpreted as version 0.
 
-* The `timestamp` attribute, if present, signifies the time at which the message was generated by its sender.
+* The `timestamp` attribute, if present,
+SHOULD signify the time at which the message was generated by its sender.
 This attribute MAY contain the Unix epoch time in nanoseconds.
 If the attribute is omitted, it SHOULD be interpreted as timestamp 0.
 
-* The `ephemeral` attribute, if present, signifies the transient nature of the message.
-For example, an ephemeral message SHOULD not be persisted by the Waku network.
-If this attribute is set to `true`, the message SHOULD be interpreted as ephemeral.
-If, instead, the attribute is omitted or set to `false`, the message SHOULD be interpreted as non-ephemeral.
+* The `ephemeral` attribute, if present,
+signifies the transient nature of the message.
+For example, an ephemeral message SHOULD not be persisted by the [64/WAKU2-NETWORK](/spec/64).
+If this attribute is set to `true`,
+the message SHOULD be interpreted as ephemeral.
+If the attribute is omitted or set to `false`,
+the message SHOULD be interpreted as non-ephemeral.
 
-# Wire Format
+### Wire Format
 
-The Waku message wire format is specified using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
+The `WakuMessage` wire format is specified using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
 
 ```protobuf
+
 syntax = "proto3";
 
 message WakuMessage {
   bytes payload = 1;
-  string content_topic = 2;
+  string contentTopic = 2;
   optional uint32 version = 3;
   optional sint64 timestamp = 10;
   optional bytes meta = 11;
   optional bool ephemeral = 31;
 }
+
 ```
 
-An example proto file following this specification can be found [here (vacp2p/waku)](https://github.com/vacp2p/waku/blob/main/waku/message/v1/message.proto).
+### Waku Message Vaildation
```

Comment:
Not sure this section belongs in this RFC. Wouldn't validation rather be a function of the relay protocol (RFC17, which already includes RLN validation)? We do a bunch of validations on Waku Messages, but this is covered in the network spec in RFC64. According to me the scope of this RFC should just be the minimal definition of a Waku Message - Relayer behaviour and how the message is interpreted otherwise belong on "higher layer" specs. Happy to hear other opinions here though.

Reply:
@jm-clius Should this spec maybe reference 64/Waku2-Network for message interpretation instead? I suggested this section so the implementer can be aware why the two attributes `contentTopic` and `payload` MUST be included in a message.